.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
We can also add a new section on Related Issues, which would be very relevant for bug fixes or feature requests filed in issues

Reply:
Maybe related issues/PRs would be more encompassing

Reply:
I think this is addressed in the Motivation and Context section, where it states that "If it fixes an open issue, please link to the issue here". Do you think that is enough?

Reply:
Yup, that sounds good enough!